src/Handlebars/Context.php

```diff
@@ -105,6 +143,19 @@ public function popIndex()
         return array_pop($this->index);
     }
 
+    /**
+     * Pop the last section data from the stack.
+     *
+     * @return array Last data
```

Comment:
Should include an @throws annotation indicating that a LogicException can be thrown.